test_runner/regress/test_unlogged.py

```diff
@@ -14,10 +14,8 @@ def test_unlogged(neon_simple_env: NeonEnv):
     conn = endpoint.connect()
     cur = conn.cursor()
 
-    cur.execute("CREATE UNLOGGED TABLE iut (id int);")
-    # create index to test unlogged index relation as well
-    cur.execute("CREATE UNIQUE INDEX iut_idx ON iut (id);")
-    cur.execute("INSERT INTO iut values (42);")
+    cur.execute("CREATE UNLOGGED TABLE iut (id int NOT NULL GENERATED ALWAYS AS IDENTITY, x int);")
```

Comment:
Hmm, so we no longer create an index. Aren't we losing test coverage of unlogged indexes?

And  I guess the reason to use `GENERATED ALWAYS AS IDENTITY` is to create a sequence? That's OK, but maybe also create and test an unlogged sequence explicitly, for clarity.

Reply:
You're right. I'll make it more explicit in the next version.